src/index.ts

```diff
@@ -120,6 +120,11 @@ type Specification = {
    * readme referenced
    */
   readonly readMePath: string
+
+  /**
+   * kind
+   */
+  readonly kind: 'EXAMPLE' | 'SWAGGER'
```

Comment:
It would be better to use enum.

Reply:
No, In TypeScript, please use union. Try to avoid enums.